src/sfctl/helps/compose.py

```diff
@@ -4,17 +4,17 @@
 # license information.
 # -----------------------------------------------------------------------------
 
-"""Help documentation for Service Fabric Docker compose related commands."""
+"""Help documentation for Service Fabric compose deployment commands."""
 
 from knack.help_files import helps
 
 helps['compose create'] = """
     type: command
     short-summary: Creates a Service Fabric compose deployment
     parameters:
-        - name: --name
+        - name: --deployment_name
```

Comment:
You need to use the CLI parameter syntax, so replace `_` with `-` here. You can double check this works by running `sfctl compose create -h` and verifying you see a description message for all parameters.